README.md

```diff
@@ -103,6 +107,109 @@ Flags:
   -h, --help   help for grant-access
 ```
 
+## Onboard
+
+Onboard everything necesary for a new application into the Operate-first environment.
+
+Creates the necesary groups, namespaces and roles, for a new project from a deployment file. Has been set up with git ops from the Operate-first/support repository, so one can raise an Onboarding-to-cluster issue and onboard that way.
+
+The onboard command expects an onboarding file, which is setup to work by default with the webhook in operate-first/support. This webhook passes in some necesary and optional parameters:
+  Necesary:
+    - Team Name
+    - Desired Project Names / Namespaces
+    - Project Description
+    - Target Cluster and Environment
+  Optional:
+    - Namespace Quota
+    - Custom Quota (per namespace)
+    - Users to add to group created
+
+For more information on this, see the onboarding template in operate-first/support docs: https://github.com/operate-first/support/blob/main/.github/ISSUE_TEMPLATE/onboarding_to_cluster.yaml
+
+The onboard command works on a configuration file which replicates the onboarding to cluster issue template listed above. This is an example of a valid configuration file using every option. Bellow each component will be discussed.
+
+```
+env: MOC
+namespaces:
+  - enable_monitoring: false
+    name: testproject
+    quota: testquota
+    disable_limit_range: false
+    project_display_name: testprojectdisplayname
+    custom_quota:
+      limits.cpu: '28'
+      requests.cpu: '28'
+      limits.memory: 32Gi
+      requests.memory: 32Gi
+      requests.storage: 100Gi
+      count/objectbucketclaims.objectbucket.io: 1
+project_description: This is the configuration for a sample project / app to onboard
+target_cluster: Smaug
+team_name: testgroup
+users:
+  - testing_gh_handle_1
+  - testing_gh_handle_2
+
+```
+
+### $target_cluster and $env
```

Comment:
> $target_cluster and $env
Remove the "$" as this is not a syntax used in the config file above so it's unclear what is being referred to. Also remove it from below and instead use `code` syntax like so: `target_cluster` and `env`